lib/client.js

```diff
@@ -1,6 +1,6 @@
-import sanityClient from "@sanity/client";
+import {createClient} from 'next-sanity'
```

Comment:
We're using the `next-sanity` toolkit, since it will let us set up real-time preview at the end.